### PR TITLE
introducing .syntax

### PR DESCRIPTION
**Pull request by hashanant**

### CODE REVIEW

File: stdplugins/_help.py [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+"""
+
+
 import sys
 from telethon import events, functions, __version__
 from uniborg.util import admin_cmd
 
 
-@borg.on(admin_cmd(pattern="helpme ?(.*)", allow_sudo=True))  # pylint:disable=E0602
+@borg.on(admin_cmd(pattern="helpme", allow_sudo=True))  # pylint:disable=E0602
```
Comment (SpEcHiDe) [inline stdplugins/_help.py]: why is the regex removed here?

File: stdplugins/_help.py [hunk 2]
```diff
@@ -1,17 +1,24 @@
...
 async def _(event):
     if event.fwd_from:
         return
-    splugin_name = event.pattern_match.group(1)
-    if splugin_name in borg._plugins:
-        s_help_string = borg._plugins[splugin_name].__doc__
```
Comment (SpEcHiDe) [inline stdplugins/_help.py]: and these lines.

File: stdplugins/_help.py [hunk 3]
```diff
@@ -22,18 +29,15 @@
...
+        results = await borg.inline_query(tgbotusername, help_string)   # pylint:disable=E0602
         await results[0].click(
             event.chat_id,
             reply_to=event.reply_to_msg_id,
             hide_via=True
         )
         await event.delete()
     else:
-        await event.reply(help_string + "\n\n" + s_help_string)
+        await event.reply(help_string)
```
Comment (SpEcHiDe) [inline stdplugins/_help.py]: ?

File: stdplugins/_help.py (hunk 1 above)
Comment (hashanant) [reply stdplugins/_help.py]: check https://t.me/ThankTelegram/259027

**Pull request merged**